python/sglang/srt/_custom_ops.py

```diff
@@ -28,7 +28,7 @@
 
 
 if not is_hip():
-    if use_vllm_custom_allreduce:
+    if use_vllm_custom_allreduce or is_hpu():
```

Comment:
if will be great if we can move these small if_statement to an env files, so we don't need to add is_hpu() to all layer/op files?

Reply:
I have taken a look at those checks. Each case is a little bit different and changing them would require larger scale change, so I prefer to address them in future PRs.